R/avg.R

```diff
@@ -55,6 +55,10 @@ avg_event <- function(id, group, par = NULL) {
       function(x) sd(x$Freq, na.rm = TRUE) / sqrt(nrow(x)),
       FUN.VALUE = numeric(1)
     )
+    count <- vapply(split(res, res$Var2),
+      function(x) nrow(x),
```

Comment:
Is `vapply(..., function(x) nrow(x))` equivalent to `vapply(..., nrow)`?

Reply:
> Is `vapply(..., function(x) nrow(x))` equivalent to `vapply(..., nrow)`?

It seems not correct. :)

Reply:
Care to show me an example when it's not equivalent? Would love to see it.

Reply:
I made a mistake. It makes!

Reply:
OK good!